lib/helpers/activeable.rb

```diff
@@ -23,3 +23,6 @@ def make_inactive
     end
   end
 end
+
+module Activeable
+end
```

Comment:
rails 6 was complaining about there not being a class named per the filename in these files. I'm not sure how to properly resolve it (a little rusty with rails) but my solution was to add a dummy one here for now to silence the error. but would like help to properly solve!